pandas/core/methods/to_dict.py

```diff
@@ -155,7 +155,7 @@ def to_dict(
             stacklevel=find_stack_level(),
         )
     # GH16122
-    into_c = com.standardize_mapping(into)
+    into_c = com.standardize_mapping(into)  # type: ignore[no-untyped-call]
```

Comment:
When we have to introduce a `# type: ignore`, we copy the error from `mypy` into here above the line.  See a few lines down in this file.